examples/domainmodel/src/cli/generator.ts

```diff
@@ -0,0 +1,79 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import fs from 'fs';
+import _ from 'lodash';
+import { CompositeGeneratorNode, IndentNode, NL, processGeneratorNode } from 'langium';
+import { AbstractElement, Domainmodel, Entity, Feature, isEntity, isPackageDeclaration, Type } from '../language-server/generated/ast';
+
+export function generateJava(domainmodel: Domainmodel, fileName: string, destination: string = '.') {
+    const path = fileName.replace(/\..*$/, '').replaceAll(/[.-]/g, '');
```

Comment:
The `replaceAll` string method is supported only by NodeJS 15, but we want to support NodeJS 12 as minimum version. Please remove all usages of that method and implement this differently.

See NodeJS compatibility here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll#browser_compatibility